packages/auth/graphql/schema.js

```diff
@@ -18,11 +18,16 @@ type queries {
   # search for users
   # required role: editor
   users(search: String!, role: String!): [User]!
+
+  # search for an unverified session by token
+  unverifiedSession(email: String!, token: String!, context: String): Session
```

Comment:
I really hate the query name, what do you guys think? @patte @tpreusse 

- getInfo is far too unspecific to get session details
- session is not true as it only returns sessions that have a token and no passport.user set

More ideas: wildSession, sessionWithToken, unclaimedSession

Reply:
IMM: `authorizeSession` for the mutation and `unauthorizedSession` for the query